src/ui/hooks/useCodePushVersion.web.test.ts

```diff
@@ -0,0 +1,10 @@
+import { renderHook } from 'tests/utils'
+
+import { useCodePushVersion } from './useCodePushVersion'
+
+describe('useCodePushVersion', () => {
+  it('should return null', () => {
```

Comment:
Pour un meilleur debug si nécessaire :
```suggestion
  it('should return null on web', () => {
```